projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-documents/documenten-api-documents.component.ts

```diff
@@ -293,7 +297,7 @@ export class DossierDetailTabDocumentenApiDocumentsComponent implements OnInit,
 
   private downloadDocument(relatedFile: DocumentenApiRelatedFile, forceDownload: boolean): void {
     this.downloadService.downloadFile(
-      `/api/v1/documenten-api/${relatedFile.pluginConfigurationId}/files/${relatedFile.fileId}/download`,
+      this.getApiUrl(`/v1/documenten-api/${relatedFile.pluginConfigurationId}/files/${relatedFile.fileId}/download`),
```

Comment:
You now solved it in the component, but can you solve this in DownloadService itself?